packages/contentstack-bootstrap/src/config.ts

```diff
@@ -50,7 +51,8 @@ const config: Configuration = {
     },
     'compass-app': {
       source: 'SunilLsagar/universal-demo',
```

Comment:
Why is this code and stacks is in your repo ?

Reply:
@shafeeqd959 Still compass app source code repo and stacks repo's are private, Once it's public will replace it.